### ISSUE TITLE
Incompatible with ktor-client 1.3.0

### ISSUE THREAD

**Issue by Djaler** [InsanusMokrassar/TelegramBotAPI#57]
```
Caused by: java.lang.IllegalStateException: Unbound [HttpClientCall] is deprecated. Consider using [request<HttpResponse>(block)] in instead.
	at io.ktor.client.call.HttpClientCallKt.call(HttpClientCall.kt:132)
	at com.github.insanusmokrassar.TelegramBotAPI.bot.Ktor.base.AbstractRequestCallFactory.prepareCall$suspendImpl(AbstractRequestCallFactory.kt:23)
	at com.github.insanusmokrassar.TelegramBotAPI.bot.Ktor.base.AbstractRequestCallFactory.prepareCall(AbstractRequestCallFactory.kt)
	at com.github.insanusmokrassar.TelegramBotAPI.bot.Ktor.KtorRequestsExecutor$execute$2.invokeSuspend(KtorRequestsExecutor.kt:41)
	at com.github.insanusmokrassar.TelegramBotAPI.bot.Ktor.KtorRequestsExecutor$execute$2.invoke(KtorRequestsExecutor.kt)
	at com.github.insanusmokrassar.TelegramBotAPI.bot.settings.limiters.EmptyLimiter.limit(EmptyLimiter.kt:4)
	at com.github.insanusmokrassar.TelegramBotAPI.bot.Ktor.KtorRequestsExecutor.execute(KtorRequestsExecutor.kt:38)
```

**Comment by InsanusMokrassar** [comment]
Hello,

In the last version of TelegramBotAPI you can find, that currently used ktor version in library is 1.2.6 (https://github.com/InsanusMokrassar/TelegramBotAPI/blob/master/gradle.properties#L7). I plan to update library near time, but can't promise anything

**Comment by InsanusMokrassar** [comment]
Fixed in 562459f0b77572e9123323c94bf035486a051277 and will be available after new version publication

**Comment by InsanusMokrassar** [comment]
Solved in #59 . @Djaler it would be great if you will check version `0.22.0`

**Comment by Djaler** [comment]
Works good, thanks!
